app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_objects = [
+        Person(person["name"], person["age"])
+        for person in people
+    ]
+    for person in people:
+        person_object = Person.people[person["name"]]
+        if "wife" in person:
+            if person["wife"]:
+                wife = Person.people[person["wife"]]
+                person_object.wife = wife
+        elif "husband" in person:
+            if person["husband"]:
+                husband = Person.people[person["husband"]]
```

Comment:
Similar to line 16, add error handling to manage cases where the husband's name is not found in the `Person.people` dictionary.